### PR TITLE
Moving contract invocation to the client side

### PR DESCRIPTION
**Pull request by alex-kooper**
Implemented client invocation on the client side, also switch sandbox port number to 8732 to be in sync with Thanos wallet.

### CODE REVIEW

File: client/src/generated/graphql_schema.tsx [hunk 1]
```diff
@@ -87,9 +87,16 @@ export type Settings = {
...
   tzStatsUrl: Scalars['String'];
   rpc: Scalars['String'];
+  admin: SettingsAdmin;
   contracts: SettingsContracts;
 };
 
+export type SettingsAdmin = {
+  __typename?: 'SettingsAdmin';
+  address: Scalars['String'];
+  secret: Scalars['String'];
```
Comment (emishur) [inline client/src/generated/graphql_schema.tsx:97]: Do not expose secret key to the client

File: server/src/schema.graphql [hunk 2]
```diff
@@ -25,9 +25,15 @@ type SettingsContracts {
...
 
+type SettingsAdmin {
+  address: String!
+  secret: String!
+}
+
 type Settings {
   tzStatsUrl: String!
   rpc: String!
+  admin: SettingsAdmin!
```
Comment (emishur) [inline server/src/schema.graphql:36]: why do you need admin settings on the client side?
You are using wallet signer now.

File: client/src/api/nftContract.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+const mkNftContract =  async (
+  tzClient: TezosToolkit, 
+  address: string, 
+): Promise<NftContract> => {
+  const contract = await tzClient.contract.at(address);
+  
+  const constants = await tzClient.rpc.getConstants();
+  const pollingInterval: number = Number(constants.time_between_blocks[0]) / 5;
```
Comment (emishur) [inline client/src/api/nftContract.ts:27]: We talked about setting polling interval when creating TezosToolkit, not for every operation

File: client/src/generated/graphql_schema.tsx (hunk 1 above)
Comment (alex-kooper) [reply client/src/generated/graphql_schema.tsx:97]: It is necessary in the current version

File: client/src/api/nftContract.ts (hunk 3 above)
Comment (alex-kooper) [reply client/src/api/nftContract.ts:27]: We also talked that doin what is done here is not easy that way.

**Comment by alex-kooper** [comment]
We will do those change after the merge

**Pull request merged**